strawberryfields/backends/tfbackend/backend.py

```diff
@@ -23,7 +23,7 @@
 
 from strawberryfields.backends import BaseFock, ModeMap
 from .circuit import Circuit
-from .ops import _check_for_eval, mixed, partial_trace, reorder_modes
+from .ops import mixed, partial_trace, reorder_modes
 from .states import FockStateTF
 
 
```

Comment:
The link in line 32 is incorrect, it is pointing to numpy, where it should point to https://www.tensorflow.org/

Reply:
Fixed!